BackofficeBundle/Resources/public/coffee/table/TableviewCollectionView.coffee

```diff
@@ -72,11 +72,9 @@ TableviewCollectionView = OrchestraView.extend(
       dom: "<'dt-toolbar'<'col-xs-12 col-sm-6'f><'col-sm-5 col-xs-6 hidden-xs'C><'col-xs-12 col-sm-1 hidden-xs'l>>"+
                     "t"+
                     "<'dt-toolbar-footer'<'col-sm-6 col-xs-12 hidden-xs'i><'col-sm-6 col-xs-12'p>>"
-      language:
-        sLengthMenu: "_MENU_"
-        sSearch: "<span class='input-group-addon'><i class='glyphicon glyphicon-search'></i></span> "
-        sInfo: "Showing <span class='txt-color-darken'>_START_</span> to <span class='txt-color-darken'>_END_</span> of <span class='text-primary'>_TOTAL_</span> entries",
-        sInfoEmpty: "<span class='text-danger'>Showing 0 to 0 of 0 entries</span>",
+      language: {
+        url: $('#contextual-informations').data('datatableTranslationUrlPattern')
```

Comment:
you should register this route in the backbone router, then call the url generator method